src/components/AirbaseNG/AirbaseNG.tsx

```diff
@@ -1,165 +1,171 @@
-     * @param {string} data - The data received from the child process.
+     * Handles incoming data from the child process.
+     * @param {string} data - Data received from the process.
      */
     const handleProcessData = useCallback((data: string) => {
-        setOutput((prevOutput) => prevOutput + "\n" + data); // Append new data to the previous output.
+        setOutput((prevOutput) => prevOutput + "\n" + data);
     }, []);
 
     /**
-     * handleProcessTermination: Callback to handle the termination of the child process.
-     * Once the process termination is handled, it clears the process PID reference and
-     * deactivates the loading overlay.
-     * @param {object} param - An object containing information about the process termination.
-     * @param {number} param.code - The exit code of the terminated process.
-     * @param {number} param.signal - The signal code indicating how the process was terminated.
+     * Handles the termination of the child process.
+     * @param {object} param - Process termination details.
+     * @param {number} param.code - Exit code of the process.
+     * @param {number} param.signal - Termination signal.
      */
     const handleProcessTermination = useCallback(
         ({ code, signal }: { code: number; signal: number }) => {
-            // If the process was successful, display a success message.
             if (code === 0) {
                 handleProcessData("\nProcess completed successfully.");
-
-                // If the process was terminated manually, display a termination message.
             } else if (signal === 15) {
                 handleProcessData("\nProcess was manually terminated.");
-
-                // If the process was terminated with an error, display the exit and signal codes.
             } else {
                 handleProcessData(`\nProcess terminated with exit code: ${code} and signal code: ${signal}`);
             }
-
-            // Clear the child process pid reference. There is no longer a valid process running.
             setPid("");
-
-            // Cancel the loading overlay. The process has completed.
             setLoading(false);
-
-            // Allow Saving as the output is finalised
             setAllowSave(true);
             setHasSaved(false);
         },
-        [handleProcessData] // Dependency on the handleProcessData callback
+        [handleProcessData]
     );
 
     /**
-     * onSubmit: Asynchronous handler for the form submission event.
-     * It sets up and triggers the airbase-ng tool with the given parameters.
-     * Once the command is executed, the results or errors are displayed in the output.
-     *
-     * @param {FormValuesType} values - The form values, containing the fake host name, channel, and WLAN interface.
+     * Handles the form submission to run the airbase-ng command.
+     * @param {FormValuesType} values - Form values.
```

Comment:
Keep the original comment. 
```suggestion
    * @param {FormValuesType} values - The form values, containing the fake host name, channel, and WLAN interface.
```